content/internships/applicant-guide/contents.lr

```diff
@@ -49,6 +50,11 @@ Please read our [Contributing Code](/contributing-code) page for information on
 
 If you're applying via Outreachy, remember to record your contribution via the Outreachy website! You can record contributions even if you have not yet received feedback on them or if they have not been merged – our mentors are busy so we can be slow to provide feedback.
 
+#### Contributing Documentation
+If you are applying for Google Season of Docs, one of the best ways you can contribute to documentation is to find places where documentation can be improved, and open an issue for it on GitHub. Be sure to state what, specifically, could be improved, and how you plan to improve it.
```

Comment:
I think it would be good to have something about substantive documentation changes to improve clarity and add information being better than typos and grammar fixes.